src/main/resources/templates/db_scripts/schema.json

```diff
@@ -31,33 +28,13 @@
         }
       ],
       "ginIndex": [
-        {
-          "fieldName": "userId",
-          "tOps": "ADD",
-          "caseSensitive": false,
-          "removeAccents": true
-        },
-        {
-          "fieldName": "itemId",
-          "tOps": "ADD",
-          "caseSensitive": false,
-          "removeAccents": true
-        },
         {
           "fieldName": "status.name",
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
         }
       ],
-      "index": [
-        {
-          "fieldName": "id",
-          "tOps": "ADD",
```

Comment:
Do not remove the lines, change `"tOps": "ADD"` to `"tOps": "DELETE"` so that the index gets removed.